pyproject.toml

```diff
@@ -84,8 +84,9 @@ tests = [
     "pytest>=8.2,<8.4",
     "pytest-cov",
     "pytest-mock",
-    "ruff==0.11.0"
-    # 'testcontainers'  # TODO: pending Postgres MR
+    "ruff==0.11.0",
+    "pytest-xdist",
+    "testcontainers"
```

Comment:
Testcontainers will be used in Postgres too. We can add it now to launch the SSH server for the integration tests.